Extracting reference data (Isolation forest).ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        model = IsolationForest(n_estimators = 500, verbose = 1, random_state = 42)
</pre><p>Another direction to experiment with that we discussed. Changing the hyperparameters, and retraining multiple times (to potentially smooth out the ref data).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>